.github/workflows/ci-enhanced.yml

```
@@ -0,0 +1,449 @@
+          cache: 'bun'
+          
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v2
+        
+      - name: Cache dependencies
+        uses: actions/cache@v4
+        with:
+          path: |
+            node_modules
+            ~/.bun/install/cache
+          key: ${{ needs.health-check.outputs.cache-key }}
```

actionlint (1.7.7)

property "health-check" is not defined in object type {build-matrix: {outputs: {}; result: string}}

(expression)